storage-tool.yml

```diff
@@ -133,6 +136,9 @@ services:
         class: 9 XB
```

Comment:
unrelated to this PR, but is `9 XB` right? what's an `XB`?

Reply:
I'm guessing this was meant to be exabyte, but that abbreviation is typically `EB`, I believe. I'm happy to change it to that.

Reply:
@mcmcgrath13 : What do you think?